docs/service-brokers/docs/020-architecture-reb.md

```diff
@@ -3,25 +3,17 @@ title: The Remote Environment Broker architecture
 type: Architecture
 ---
 
-The Remote Environment Broker (REB) workflow starts with the registration process, during which a remote environment (RE) is registered on the Kyma cluster.
+The Remote Environment Broker (REB) workflow consists of the following steps:
 
-### Remote environment registration process
+1. The Remote Environment Broker watches for RemoteEnvironments (REs) in the cluster and EnvironmentMappings (EMs) in all Namespaces.
+2. The user creates an EnvironmentMapping custom resource in a given Environment. The EnvironmentMapping activates the services offered by a given RemoteEnvironment and has the same name as the RemoteEnvironment it corresponds to.
```

Comment:
`The EnvironmentMapping activates the services offered by a given RemoteEnvironment and has the same name as the RemoteEnvironment it corresponds to.`

`The EnvironmentMapping activates the services offered by a given RemoteEnvironment which has the same name.`